.github/workflows/chromatic.yaml

```diff
@@ -0,0 +1,80 @@
+# Chromatic action from https://www.chromatic.com/docs/github-actions
+
+# Workflow name
+name: "chromatic"
+
+on:
+  issue_comment:
+    types: [created]
+
+# List of jobs
+jobs:
+  visual-testing:
+    # Operating System
+    runs-on: ubuntu-latest
+
+    # Job steps
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # 👈 Required to retrieve git history
+
+      - uses: 8BitJonny/gh-get-current-pr@1.1.0
+        id: getPr
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          # Only return if PR is still open
+          filterOutClosed: true
+
+      - run: echo "The current PR number is ${prNumber}"
+        if: success() && steps.getPr.outputs.number
+        env:
+          prNumber: ${{ steps.getPr.outputs.number }}
+
+      - name: Get most recent comment
+        if: success() && steps.getPr.outputs.number
+        uses: peter-evans/find-comment@v1
+        id: latestComment
+        with:
+          issue-number: ${{ steps.getPr.outputs.number }}
+          body-includes: ""
+          direction: last
+
+      - run: |
+          echo "runChromatic=${runChromatic}" >> $GITHUB_ENV
+        env:
+          runChromatic: ${{ contains(steps.latestComment.outputs.comment-body, 'run chromatic') }}
+
+      - run: echo "found run chromatic text in latest comment"
+        if: ${{ env.runChromatic == 'true' }}
+
+      - uses: actions/cache@v2 # use of cache for node_modules saves usually > 1min of build time
+        if: ${{ env.runChromatic == 'true' }}
+        with:
+          path: "**/node_modules"
+          key: ${{ runner.os }}-${{ hashFiles('yarn.lock') }}-${{ hashFiles('**/package.json') }}
```

Comment:
I might be misunderstanding something, but why do we need to store this information (`key`)?

Reply:
I'm not really sure, that was copy pasta from the unified-home config.

Reply:
Ah! Okay looking at unified's config and these docs it looks like a pretty standard way to cache deps: https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows